README.md

```diff
@@ -1,2 +1,63 @@
-# hello-world-project
-Hello World Project - Day 2 - Week 01
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="logo.png" alt="logo" width="140" height="auto" />
+  <br/>
+  <h3><b>Microverse First Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+# 📖 Hello World Microverse <a name="about-project"></a>
+
+**Hello World Microverse** is our first project. We were practicing all related about: Git/GitHub, Linters, using Templates and a few of HTML/CSS.
+
+## 🛠 Built With Visual Studio Code and NPM<a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+We built this project using tools as: Git client, Visual Studio Code and NPM.
+
+### Key Features <a name="key-features"></a>
+
+We developed this exercise to test some skills using Git/GitHub and Linters.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This job was suggested only for educative purposes.
+
+### Prerequisites
+
+In order to run this project you only need a web browser. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make sure, in your README you have all the sections from the provided template which apply to your project and filled with infos accordingly like:
**Stacks**: `You may consider listing HTML and CSS since are the stacks you have used`
**Getting started**: `You may consider giving some commands on how to get the remote repos all the way down in your local environment, how to navigate on it.`
**Usage**: `you may consider providing some commands or procedures on how to use or run the project...`

Reply:
Hey, @christianbiring1 thanks for your suggestions, buddy. 

In a few words I need to:

1.- Write a summary of all the work made in my Pull Request.
2.- Change the name of my branch "features" for another one (like "Hello Microverse").
3.- Fix in my Readme template the three suggested sections (Stacks, Getting Started, and Usage).

Reply:
Hey @jlcarrascof,
Yeah sure, in the box after the title you need to write the summary describing the work you have done in the PR,
You may also consider changing the branch name to `hello-microverse` and in your README add the missing sections as suggested and complete the ones which weren't already.

Reply:
Yeah sure, by the way any time you have some issues you can always reach out to me on slack, I use the same name there.

Reply:
Good morning @christianbiring1 when you read this.

I was updating all your suggestions during the day, I think that it's done. Please, check it when you have time and share your feedback, man. Thanks in advance